.rspec

```diff
@@ -1,2 +0,0 @@
---color
```

Comment:
Fixed.  Personally I like --format documentation as well because it acts like...uh...documentation and helps to show whether or not the design is reasonably coherent.